### PR TITLE
bumps to 3.1.2 - updates primary nav breakpoints to delay hamburger …

### PR DESCRIPTION
**Pull request by schweitzdesign**
…until 960px

### CODE REVIEW

**Comment by katywatkins** [comment]
Overall I think this is a good step forward, adjusting the breakpoint without significantly altering the rest of the nav. Just some notes/questions around detail stuff to follow.

**Comment by katywatkins** [comment]
under 1200px the divider lines in the account nav disappear, is that intentional?
screen shot 2017-02-07 at 1 50 12 pm
screen shot 2017-02-07 at 1 50 44 pm

**Comment by katywatkins** [comment]
at 960px-991px, the divider lines on the help menu disappear (nicely reversing the existing bug in production, where those lines are briefly visible in the collapsed version)
screen shot 2017-02-07 at 1 57 20 pm
screen shot 2017-02-07 at 1 58 04 pm

**Comment by katywatkins** [comment]
Since we're messing with the nav already, what are your thoughts on implementing a quick and dirty fix for the overflow issue in the account dropdown that pushes 'log out' off?

screen shot 2017-02-07 at 2 00 59 pm

I poked at this a bit back when we first got the ticket for it, turns out ellipsizing a multi-line block of text isn't as easy as a single line. 😭  The quickest and most complete solution I had so far was just setting that h4 to a max height of 48px. It's not ideal, but it seems better than feeling like you've lost the ability to log out entirely.
screen shot 2017-02-07 at 1 30 28 pm

File: app/assets/stylesheets/forever_style_guide/modules/_nav-icons.scss [hunk 1]
```diff
@@ -140,24 +146,23 @@ a.cart-icon {
   min-width: $cart-count-width;
   padding-top: 1px; // vertical align number in circle
```
Comment (katywatkins) [inline app/assets/stylesheets/forever_style_guide/modules/_nav-icons.scss]:
```
@media (min-width: $grid-float-breakpoint-max) and (max-width: $screen-lg) {
    padding-top: 0;
}
```
Not sure if this is the best fix, but the positioning of the cart count seems off between those breakpoints
screen shot 2017-02-07 at 3 55 20 pm  screen shot 2017-02-07 at 3 56 22 pm

**Comment by katywatkins** [comment]
.dropdown-menu-nav-spacer doesn't quite cut it below 1200px, but that's also about the same amount of variation in the 'get started' columns that we have in the web app version currently
screen shot 2017-02-07 at 3 00 08 pm

File: app/views/forever_style_guide/sections/components/navigation/_nav_account_dropdown_menu.erb [hunk 2]
```diff
@@ -44,13 +44,13 @@
           </div>
         </li>
 
-        <% if current_user.linked_ambassador.try(:active?) %>
+        <% if !current_user.linked_ambassador.try(:active?) %>
```
Comment (katywatkins) [inline app/views/forever_style_guide/sections/components/navigation/_nav_account_dropdown_menu.erb]: I don't think you meant to commit this, it's hiding the back office link for current ambassadors

**Comment by schweitzdesign** [comment]
Whoa! Good catch. Update now.

**Comment by katywatkins** [comment]
LGTM!

**Pull request merged**